CHANGELOG.adoc

```diff
@@ -7,9 +7,9 @@
 // the empty line separators.
 
 ////
-| 🎁🐛🐣🗑️✨
-|
-| https://github.com/knative/client/pull/[#]
+| 🐛
+| Updated to work with Go 1.20
+| https://github.com/knative/client/pull/1775[#1775]
```

Comment:
Oops, I just saw that the Changelog is quite outdated 😬 (@dsimansk would it make sense to bring it up to date, covering the releases in the second half of 2022 ?)

However, @scottmason88 can you please keep the template text and move your part out of the comment (`//// .... ////`) ? Also you should add a new table for the 1.10 release (the next release), without a date yet. Then add your entry as the first row. That would be great !

Reply:
Well, we don't use it anymore. I should have added a deprecation notice to that file probably. We're using release script generated release notes.

Reply:
I'd skip the update from this PR.

Reply:
I removed my changes to the changelog so that the comment is still there.